### PR TITLE
PEP 588 Updates

### PR DESCRIPTION
**Pull request by Mariatta**
- hire a professional project manager
- create playground issue tracker for experiment
- test repo for labels
- python triage team has been created
- devguide update

### CODE REVIEW

File: pep-0588.rst [hunk 1]
```diff
@@ -24,6 +24,17 @@ Here we outline the tasks, steps, and core decisions we need to make
...
+-----------------------------------
+
+Having a professional project manager to handle the migration, similar to how
+the Warehouse project was managed, would help ensure the success of this project.
+
+Create a playground CPython issue tracker on GitHub
+---------------------------------------------------
+
+We should create a playground issue tracker on GitHub where we can experiment
+test out the new workflow.
```
Comment (willingc) [inline pep-0588.rst]:
```suggestion
and test out the new workflow.
```

File: pep-0588.rst [hunk 2]
```diff
@@ -65,10 +79,17 @@ It has been `proposed on Discourse
...
 issues, notifying the appropriate parties, as well as applying labels
-to issues and pull requests. We can grant the "write" permission to
-the "bug triage" team, while limiting merging pull requests to
-"CPython core developer" team on GitHub.
+to issues and pull requests.
 
+The new Triage role on GitHub is currently in beta, and the Python organization
+has been granted access to this role, and we can begin taking advantage of it.
+
+The "Python Triage" team has been created. Description and expectation of the
```
Comment (willingc) [inline pep-0588.rst]:
```suggestion
The "Python Triage" team has been created. A description and expectations of the
```

File: pep-0588.rst [hunk 3]
```diff
@@ -65,10 +79,17 @@ It has been `proposed on Discourse
...
+triage role `has been added to Devguide <https://devguide.python.org/triaging/#python-triage-team>`_.
```
Comment (willingc) [inline pep-0588.rst]:
```suggestion
triage role `have been added to Devguide <https://devguide.python.org/triaging/#python-triage-team>`_.
```

**Comment by Mariatta** [comment]
Thanks @willingc !

**Pull request merged**